test/test_07_03.py

```diff
@@ -0,0 +1,18 @@
+import numpy as np
+
+from hm_1.tasks.task_07_03 import tic_tac_toe_checker
+
+
+def test_tic_tac_toe_returns_unfinished():
+    board = np.array([["-", "-", "o"], ["-", "o", "o"], ["-", "x", "x"]])
+    assert tic_tac_toe_checker(board) == "unfinished"
+
+
+def test_tic_tac_toe_returns_o_wins():
```

Comment:
We have multiple ways to win the match.  Lest group these cases into categories. How many win categories we can have (how many tests should we add) ?

Reply:
8 with x and 8with o?

Reply:
There are 16 options to win.  But do we really need to test them all?  We could group some of them together.

The group is a couple of win situations that very similar and we can test just one of these. It is easy to find them if you look into your code, usually, all members of the group are processed in the same way. 

There are  2 ways how to group these outputs. One you have already mentioned (x and o) and now need to find the other.

Why do we need that?  Because this number of groups is the actual number of tests for winning situation you need to write.

`total_tests_count = group_count * xxx + 1 * ooo`. Where `1 < group_count < 8` (1 is not enough, 8 is too much)